src/graphics/opengl/RendererGL.cpp

```diff
@@ -603,11 +597,19 @@ namespace Graphics {
 	{
 		PROFILE_SCOPED()
 		//same as above
-		SetMatrixMode(MatrixMode::MODELVIEW);
-		LoadMatrix(m);
+
+		// SetMatrixMode(MatrixMode::MODELVIEW);
```

Comment:
Remove instead of comment out

Reply:
This is that dead code elimination I was talking about - forgot to clean this up, but I still have one more commit to go, I'll clean it up then.